src/arrayMethodUnshift.js

```diff
@@ -5,7 +5,17 @@
  */
 function applyCustomUnshift() {
   [].__proto__.unshift2 = function(...elements) {
-    // write code here
+    const unshiftedArray = [...elements, ...this];
+
+    if (elements.length === 0) {
```

Comment:
this can be placed before all other action

Reply:
Sorry, my bad